crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1225,7 +1225,7 @@ where
     #[instrument(level = "trace", skip_all, target = "consensus::engine", ret)]
     fn try_buffer_payload(
         &mut self,
-        block: SealedBlock,
+        block: SealedBlock<HeaderTy<N>, BodyTy<N>>,
```

Comment:
I believe we can use 

https://github.com/paradigmxyz/reth/blob/0daa456f3a93ccb7d02dd6e5d617eaef7d1ea466/crates/primitives/src/block.rs#L487-L487
for all `SealedBlock` `SealedBlockFor<N>`

Reply:
I pushed this change just as a sample. Is this what you're referring to?
Because I've tried using `SealedBlockFor<N>` but `N` needs to implement `reth_primitives::Block`.

```rust
    fn try_buffer_payload(
        &mut self,
        // block: SealedBlock<HeaderTy<N>, BodyTy<N>>,
        block: SealedBlockFor<reth_primitives::Block>,
    ) -> Result<PayloadStatus, InsertBlockError> {
        self.blockchain.buffer_block_without_senders(block)?;
        Ok(PayloadStatus::from_status(PayloadStatusEnum::Syncing))
    }
```

I hope it makes sense. I'm trying to figure out the different types in play here.
Thanks!